AmazonChimeSDKDemo/AmazonChimeSDKDemo/MeetingModel.swift

```diff
@@ -123,7 +120,7 @@ class MeetingModel: NSObject {
     }
 
     var isLocalVideoActive = false {
-        didSet {
+        willSet(isLocalVideoActive) {
```

Comment:
We dont narmally use willSet, is this necessary?

Reply:
Yes, it will not fix the issue without it.  This makes it only call `startLocalVideo` if it is switching from false to true, not true to true.